### PR TITLE
WordPress and PhpStorm

### PR DESCRIPTION
**Pull request by alexfornuto**
Addresses 1/2 of #2616 
Supersedes #2627 

## Effect
PR includes the following changes:
- Update document to latest version of Php storm, and edit for clarity and pitfalls.
- Rebrands the guide as being best practices when using Pantheon develop custom plugins and themes.
-

## Remaining Work
- [x] Copy review.

### CODE REVIEW

File: source/_docs/wordpress-phpstorm.md [hunk 1]
```diff
@@ -1,64 +1,97 @@
...
+description: Configure a local environment for building a WordPress plugin or theme using JetBrains PhpStorm.
 tags: [local]
 categories: [wordpress]
 ---
 ## Overview
 
 [JetBrains PhpStorm](https://www.jetbrains.com/phpstorm/) is a commercial PHP IDE that can be configured to work with WordPress sites, allowing users to easily build and maintain custom plugins.
 
-For detailed information, see [WordPress Development using PhpStorm](https://confluence.jetbrains.com/display/PhpStorm/WordPress+Development+using+PhpStorm). This document focuses on integrating and showcasing the PhpStorm workflow.
+This doc will set you up with a local environment in PhpStorm to create your custom plugins and themes. If you're looking to use PhpStorm to manage code for an entire site, [click here](/docs/wordpress-phpstorm/).
```
Comment (rachelwhitton) [inline source/_docs/wordpress-phpstorm.md]: This seems wrong, the page shouldn't link to itself.

File: source/_docs/wordpress-phpstorm.md [hunk 2]
```diff
@@ -1,64 +1,97 @@
...
+
+For detailed information, see [WordPress Development using PhpStorm](https://confluence.jetbrains.com/display/PhpStorm/WordPress+Development+using+PhpStorm). This document focuses on integrating and showcasing the PhpStorm workflow and is current as of PhpStorm version 2017.1.4.
```
Comment (rachelwhitton) [inline source/_docs/wordpress-phpstorm.md]: We should rewrite this overview section to clarify the value and objective of this page. It's not clear why I would need to follow a different guide if I want to manage code for an entire site. Why would I want to manage custom code different or separately?

File: source/_docs/wordpress-phpstorm.md [hunk 3]
```diff
@@ -1,64 +1,97 @@
...
+
+## Create a WordPress site
+
+If you do not already have one, [create a new WordPress site](/docs/launch-wordpress/) on Pantheon.
```
Comment (rachelwhitton) [inline source/_docs/wordpress-phpstorm.md]: Is this site intended to be used exclusively for general plugin/theme development?

File: source/_docs/wordpress-phpstorm.md [hunk 4]
```diff
@@ -1,64 +1,97 @@
...
+
+1. Type: `SFTP`
+1. SFTP Host: `appserver.dev.<Site UUID>.drush.in`
+1. Port: `2222`
+1. User name: `dev.<Site UUID>`
+1. Auth Type: Select Password and specify password, or key pair and select your private SSH key.
+1. Root Path: Click the **Autodetect** button and the Root Path should be automatically populated:
+
+    ![Add web server PhpStorm](/source/docs/assets/images/phpstorm-deployment-connection-tab.png)
+    <div class="alert alert-info" role="alert">
```
Comment (rachelwhitton) [inline source/_docs/wordpress-phpstorm.md]: Suggest `alert-danger` class and Warning h4 instead of note

File: source/_docs/wordpress-phpstorm.md [hunk 5]
```diff
@@ -1,64 +1,97 @@
...
+2. Deployment path on server: `/code`
+3. Web path on server: `/`
 4. Click **OK**.
 
-Now go to Tools > Deployment > Automatic Upload > and select the server you created. Any files you change and save locally will be automatically uploaded to Pantheon.
+## Enabling Automatic Upload
+
+Go to **Tools Menu** > **Deployment** > **Automatic Upload**.
+
+You are now ready to begin editing your code. Any files you change and save locally will be automatically uploaded to your Pantheon development environment.Once your code is complete, you can deploy your work into Test and then ultimately to Live from your Pantheon Dashboard.
```
Comment (rachelwhitton) [inline source/_docs/wordpress-phpstorm.md]: We may need to change this conclusion based on the overview rewrite and the clarified objective. If the purpose of this guide is to set up a site to be used for general development, and it's not a site intended to be launched, it wouldn't align with the conclusion here.

File: source/_docs/wordpress-phpstorm.md (hunk 1 above)
Comment (alexfornuto) [inline source/_docs/wordpress-phpstorm.md]: DERP!

File: source/_docs/wordpress-phpstorm.md (hunk 3 above)
Comment (alexfornuto) [inline source/_docs/wordpress-phpstorm.md]: Yes?

File: source/_docs/wordpress-phpstorm.md (hunk 4 above)
Comment (alexfornuto) [inline source/_docs/wordpress-phpstorm.md]: ![bitmoji](https://render.bitstrips.com/v2/cpanel/10117701-243506170_5-s4-v1.png?transparent=1&palette=1&width=246

**Comment by alexfornuto** [comment]
@rachelwhitton I believe I've addressed all your concerns?

**Pull request merged**